src/Components/CreateImageWizard/steps/FileSystem/components/MinimumSize.tsx

```diff
@@ -7,40 +7,46 @@ import {
 } from '../../../../../store/wizardSlice';
 import { useFilesystemValidation } from '../../../utilities/useValidation';
 import { ValidatedInputAndTextArea } from '../../../ValidatedInput';
-import { Partition } from '../fscTypes';
+import { FilesystemPartition, LogicalVolumeWithBase } from '../fscTypes';
 
 type MinimumSizePropTypes = {
-  partition: Partition;
+  partition: FilesystemPartition | LogicalVolumeWithBase;
+  customization: 'disk' | 'fileSystem';
 };
 
-const MinimumSize = ({ partition }: MinimumSizePropTypes) => {
+const MinimumSize = ({ partition, customization }: MinimumSizePropTypes) => {
   const dispatch = useAppDispatch();
   const stepValidation = useFilesystemValidation();
 
   return (
     <ValidatedInputAndTextArea
       ariaLabel='minimum partition size'
-      value={partition.min_size}
+      value={partition.min_size || ''}
       isDisabled={partition.unit === 'B'}
       warning={
         partition.unit === 'B'
           ? 'The Wizard only supports KiB, MiB, or GiB. Adjust or keep the current value.'
           : ''
       }
       type='text'
+      placeholder='Define minimum size'
       stepValidation={stepValidation}
       fieldName={`min-size-${partition.id}`}
-      placeholder='File system'
```

Comment:
Please don't ask me how long it took me to figure out the tests are tripping on missing placeholder I've removed a few commits earlier :expressionless: